docs/src/use/configure/configuration-files.md

```diff
@@ -495,3 +498,136 @@ npx eslint --config some-other-file.js **/*.js
 ```
 
 In this case, ESLint does not search for `eslint.config.js` and instead uses `some-other-file.js`.
+
+## TypeScript Configuration Files
+
+::: warning
+This feature is currently experimental and may change in future versions.
+:::
+
+To enable TypeScript configuration files, you must install the optional dev dependency [`jiti`](https://github.com/unjs/jiti) in your project.
```

Comment:
I suppose we should also point out that this step is unnecessary except in Node.js. So maybe something more like this:
```suggestion
For Deno and Bun, TypeScript configuration files are natively supported; for Node.js, you must install the optional dev dependency [`jiti`](https://github.com/unjs/jiti) in your project (this dependency is not automatically installed by ESLint):
```